editor/utils/mobile.js

```diff
@@ -0,0 +1,13 @@
+import { BREAK_MEDIUM } from '../constants';
+
+/**
+ * Disables isSidebarOpened on rehydrate payload if the user is on a mobile screen size.
+ *
+ * @param  {Object}  payload   rehydrate payload
+ * @param  {Boolean} isMobile  flag indicating if executing on mobile screen sizes or not
+ *
+ * @return {Object}            rehydrate payload with isSidebarOpened disabled if on mobile
+ */
+export const disableIsSidebarOpenedOnMobile = ( payload, isMobile = ( window && window.innerWidth < BREAK_MEDIUM ) ) => (
```

Comment:
If `window` was not declared, I expect this would throw an error, same as:

```js
foo && foo.bar
// VM124:1 Uncaught ReferenceError: foo is not defined at <anonymous>:1:1
```

This is where an `'undefined' !== typeof window` could be used. If you do this, I'd suggest not assigning this as the fallback parameter in arguments for readability's sake, and either moving it into a top-level variable, or performing the logic within the function itself.